internal/k8s/reconciler/listener_test.go

```diff
@@ -537,7 +537,7 @@ func TestListenerCanBind_RouteKind(t *testing.T) {
 	routeMeta.SetGroupVersionKind(schema.GroupVersionKind{
 		Group:   gwv1alpha2.GroupVersion.Group,
 		Version: gwv1alpha2.GroupVersion.Version,
-		Kind:    "UDPRoute",
+		Kind:    "TCPRoute",
```

Comment:
Simply needed a substitue for the test, and the `Kind` here only needed to be incompatible with the protocol below, `http`